.github/workflows/e2e.yml

```diff
@@ -0,0 +1,118 @@
+name: e2e
+
+on:
+  push:
```

Comment:
Do you expect both push and pull-request events?

Reply:
No, I don't. I expect only pushed the main branch. In short, I want to run e2e tests after merging to the main branch. 

I think we test `python-package` tests so don't need to run e2e test every pull-request.

Reply:
To be clear, this Actions will be triggered when

* We directly push commits to the main branch
* We merge PRs to the main branch

If we only want to run Actions when we merge PRs to the main branch, we need to specify on.pull_request event with if: github.event.pull_request.merged == true 📝

Reply:
> with if: github.event.pull_request.merged == true

TIL, Thank you!
But if someone pushes to the main branch directly, we should check it. So, I'll keep it.